create_tag.py

```diff
@@ -85,6 +85,36 @@ def autoincrement_version(latest_version, semver=False, semver_bump_type="major"
         return str(version.major + 1)
 
 
+def get_full_username(api, username):
+    """
+    Return the full name of a github user
+    """
+    user = api.users.get_by_username(username=username)
+
+    return user['name']
+
+
+def list_reviewers_for_pr(api, repo, pr_number):
+    """
+    Return all reviewers that approved a pull request
+    """
+    try:
+        res = api.pulls.list_reviews(owner="osbuild",repo=repo, pull_number=pr_number, per_page=20)
+    except:
+        msg_info(f"Couldn't get reviewers for PR #{pr_number}.")
+        res = None
+
+    reviewers = ""
+
+    if res is not None:
+        for item in res:
+            if item['state'] == "APPROVED":
+                reviewer = get_full_username(api, item['user']['login'])
+                reviewers += f"{reviewer} "
+
+    return reviewers.strip()
```

Comment:
When there are multiple reviewers who approved the PR, their names get concatenated without any delimiter other than a space, which makes it harder to determine their names (some people may have middle name).

e.g. from running this on `osbuild` repo:

```
Changes with 96

----------------
  * stages/ostree.deploy: drop requirement on rootfs option (#1385, Author: Dusty Mabe, Reviewers: Colin Walters Achilleas Koutsou)

— Somewhere on the Internet, 2023-09-14
```

I propose to make `reviewers` a list `[]` and append their names to it in the `for` loop.

The caller will then be able to `', '.join(reviewers)`

Reply:
Proposed a solution to this now, including setting a fallback in case there was no reviewer (hopefully rarely/never useful :crossed_fingers: )